src/ray/gcs/tables.cc

```diff
@@ -112,6 +112,19 @@ template <typename ID, typename Data>
 Status Log<ID, Data>::Subscribe(const JobID &job_id, const ClientID &client_id,
                                 const Callback &subscribe,
                                 const SubscriptionCallback &done) {
+  auto subscribeWrapper = [subscribe](AsyncGcsClient *client, const ID &id,
```

Comment:
`subscribeWrapper` seems not consistent with the code style. May be `subscribe_wrapper`?

Reply:
thx!